RunningOrder/Sprint/Sprint.swift

```diff
@@ -7,9 +7,12 @@
 //
 
 import Foundation
+import SwiftUI
```

Comment:
I think we should avoid SwiftUI in the model as it is not part of the UI. maybe keep only the color code, or RGB in order to create SwiftUI color with it ? @bilalb  ?

Reply:
if the color is stored in the xcassets we can only keep the string identifier

Reply:
yep ! can you apply this change ? :)